docs/loader_usage.md

```diff
@@ -0,0 +1,114 @@
+# Scylla Cluster Tests (SCT): Loaders and Stress Commands
+
+This document explains how to configure and use loaders and stress commands effectively in Scylla Cluster Tests (SCT). It primarily focuses on Longevity Tests (`longevity_test.LongevityTest.test_custom_time`), although the concepts apply similarly to other tests, which might include additional parameters (refer to specific test code/documentation for details).
+
+---
+
+## Overview of Loaders
+
+Loaders are specialized machines used to generate workloads on Scylla clusters. They run stress tools such as:
+
+- [cassandra-stress](https://github.com/scylladb/cassandra-stress/) - the most commonly used stress tool, simulating various workloads on Scylla with predefined columns count, row size and user profiles for custom ones.
+- [scylla-bench](https://github.com/scylladb/scylla-bench/) - a newer stress tool designed for Scylla, providing more advanced features especially for testing large partitions.
```

Comment:
Let's remove the word "newer".
It is old enough.

Main point is that it is useful for testing large partitions and go driver.